Nunit_proj/Pages/EducationNegativeTab.cs

```diff
@@ -0,0 +1,95 @@
+﻿using NUnit.Framework;
+using Nunit_proj.Utilities;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Nunit_proj.Pages
```

Comment:
you should avoid creating pages by test cases e.g happy or negative! your page should only be named around the actual feature. for future please rename this accordingly for now its okay